celavi/pylca_celavi/pylca_opt_foreground.py

```diff
@@ -16,7 +16,7 @@
 
 
 #We are integrating static lca with dynamics lca over here. 
-def preprocessing(year,df_static,dynamic_lci_filename):
+def preprocessing(year,state,df_static,dynamic_lci_filename):
```

Comment:
I think we'll probably want to compare results using the national grid to the state grid at some point. Can you add some functionality to use either grid mix, either through a Boolean flag or by defaulting to the national grid if the grid mix file doesn't have a state column?

Reply:
I have started implementing this functionality - But there are some issues due to many parts of the code requiring change......Working through them.

Reply:
Incorporated